RecoTauTag/Configuration/python/loadRecoTauTagMVAsFromPrepDB_cfi.py

```diff
@@ -41,6 +41,8 @@
 }
```

Comment:
diff too is not being too smart here.
I would assume this only loads new items compared to what's currently in the release, can you confirm?

Reply:
Yes, I confirm that it only adds new 2017v2 payloads. In particular, it means that standard RECO and MiniAOD v1 and v2 workflows are not broken.